apps/central-scan/frontend/README.md

```diff
@@ -1,7 +1,7 @@
 # VotingWorks Ballot Scanner (BSD) (VxCentralScan)
 
-Scans ballots printed by the VxSuite [Ballot Marking Device (BMD)](../bmd) or
-the VxSuite [Election Manager](../election-manager).
+Scans ballots printed by the VxSuite [Ballot Marking Device (BMD)](../mark) or
+the VxSuite [Election Manager](../admin).
```

Comment:
Thank you for checking on our READMEs! This was very out of date. We don't even have ballot printing enabled from `../admin` anymore, so I suggest replacing this header + tidbit with:

```
# VotingWorks Batch Scanner (VxCentralScan)

Scans ballots in batches using COTS (Commercial-off-the-Shelf) Fujitsu scanners.
```

Reply:
I think the paths also need to go back another directory now